src/FriendsPage.tsx

```diff
@@ -1,18 +1,83 @@
 import React, { useState, useEffect, useMemo } from "react";
-import { Friend, FriendStatus } from "./ourtypes";
+import { Friend } from "./ourtypes";
 import PageLayout from "./PageLayout";
 import AutoTable, { TableHeader } from "./Table";
+import { Button } from "react-bootstrap";
 
 const FriendsPage: React.FC = () => {
   const [error, setError] = useState<any>();
   const [isLoaded, setIsLoaded] = useState(false);
   const [friends, setFriends] = useState<Friend[]>([]);
   const [nearbyPeople] = useState<Friend[]>([]);
 
+  interface ButtonGroupProps {
+    id: number
+  }
+  
+  const RequestResponseButtonGroup: React.FC<ButtonGroupProps> = ({ id }) => {
+    return (
+      <div>
+        <Button
+          type="button"
+          variant="primary"
+          size="sm"
+          onClick = {() => { AcceptFriendship(id) }}
+        >Accept</Button>{' '}
+        <Button
+          type="button"
+          variant="outline-danger"
+          size="sm"
+          onClick = {() => { RejectFriendship(id) }}
+        >Reject</Button>
+      </div>
+    );
+  }
+
+  function AcceptFriendship(id: number) {
+    setIsLoaded(false);
```

Comment:
I don't think setting `loaded` is necessary. Changes you make to the list `friends` with cause the the page to re-render with the new data I believe.